print_sorted method called with 1 arg:
	>>> ml.print_sorted([4, 2, 5])
	Traceback (most recent call last):
	TypeError: print_sorted() takes 1 postional argument but 2 were given

print_sorted method called with 2 args:
	>>> ml.print_sorted([4, 2, 5], 1)
	Traceback (most recent call last):
	TypeError: print_sorted() takes 1 positional argument but 3 were given

Empty list_test:
	>>> ml = MyList()
	>>> ml.print_sorted()
	[]

Normal list test:
	>>> ml = MyList([3, 1, 2])
	>>> ml.print_sorted()
	[1, 2, 3]

Negative ints list test:
	>>> ml = MyList([-2, -4, -8])
	>>> ml.print_sorted()
	[-8, -4, -2]
